frontend/src/metabase/common/utils/column-groups.ts

```diff
@@ -18,5 +13,5 @@ export function getColumnGroupIcon(
   if (groupInfo.isImplicitlyJoinable) {
     return "connections";
   }
-  return;
+  return "sum";
```

Comment:
why "sum"? it wasn't like this before

Reply:
If we get here, it means it's a summarization group